src/openapi-vuepress-markdown.ts

```diff
@@ -5,6 +5,14 @@ import { Command } from 'commander'
 import { generateMarkdownFiles } from './markdownAdapters'
 const SwaggerParser = require('swagger-parser')
 
+function validateNumber(arg: string, name: string): boolean {
+    if (arg && isNaN(Number(arg))) {
```

Comment:
Do we explicitly want to allow whitespace in the numbers and non-decimal numbers here?

Reply:
Not sure I understand.

Reply:
What I mean is this is permitting inputs that we likely don't want permitted:


```
> isNaN(Number("0x123"))
false
> isNaN(Number("123 "))
false
> isNaN(Number("0o123"))
false
> isNaN(Number("0b111"))
false
```

Reply:
Fixed @jstasiak

Reply:
Cheers, sorry for being unclear